src/main/java/model/events/Training.java

```diff
@@ -1,10 +1,13 @@
 package model.events;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
 
 import model.persons.Trainer;
 
 public class Training {
+	public static final List <Training> trainings = new ArrayList <>();
```

Comment:
Please take a look at the UML diagram. I think it is not good to implement the list with all the trainings in the Training-class itself. Better take this to TrainerController?